test/fixtures/trip_dismissals.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
Turn this into some real fixtures based on the students and trips in the other fixtures